mongomock/collection.py

```diff
@@ -963,6 +926,59 @@ def _copy_only_fields(self, doc, fields, container):
                 fields[field] = op
             return doc_copy
 
+    def _combine_projection_spec(self, projection_fields_spec):
```

Comment:
You don't use `self` here so it's better to make it a function rather than a method.